xbmc/pvr/PVRManager.cpp

```diff
@@ -636,6 +648,9 @@ bool CPVRManager::LoadComponents(CPVRGUIProgressHandler* progressHandler)
   if (progressHandler)
     progressHandler->UpdateProgress(g_localizeStrings.Get(19236), 0); // Loading channels from clients
 
+  if (!m_providers->Load() || !IsInitialising())
+    return false;
```

Comment:
The thought was that in a follow on PR and when reworking Channel Groups, we could also have groups representing the providers. So we load them first to make that possible. If we chose to put that on the add-on and have no code for that in Kodi it could be loaded anytime. But I think it would be nice to be able to have "providers groups for free". What do you think?

Reply:
Keep it for now. We will see whether this actually has negative impact on PVR startup.